src/common/services/session/sessionModal.service.js

```diff
@@ -91,9 +96,6 @@ export default angular
   .factory(serviceName, SessionModalService)
   .config(function (modalStateServiceProvider) {
     modalStateServiceProvider.registerModal(
-      'reset-password',
       /* @ngInject */
-      function (sessionModalService) {
-        sessionModalService.resetPassword()
-      })
+      function (sessionModalService) {})
   })
```

Comment:
@dr-bizz Can we remove this `config` block? It doesn't seem like it's doing anything.

Reply:
Yes, we can remove it. It was ensuring the forgot-password modal appeared whenever reset-password was triggered. But as we no longer use that modal, we can remove it.